src/blogtoblog/import.ts

```diff
@@ -143,7 +153,8 @@ async function main() {
   //   },
   // ];
 
-  const entries = await getEntries();
+  const allEntries = await getEntries();
+  const entries = sectionData(allEntries, argMin, argMax);
```

Comment:
I understood why you are doing here but it is a good practice to not mix several things in the same PR: if we have to revert the PR, this code will be gone too.